client/app/pages/users/UsersList.jsx

```diff
@@ -0,0 +1,248 @@
+import { extend, map, filter } from 'lodash';
+import moment from 'moment';
+import React from 'react';
+import { react2angular } from 'react2angular';
+import classNames from 'classnames';
+import TimeAgo from 'react-timeago';
```

Comment:
Maybe `moment`'s Duration can be of use here https://momentjs.com/docs/#/durations/humanize/

Reply:
This component (as well as `am-time-ago` which we use in Angular code) is interactive. Do you think it's better to implement own component instead of `react-timeago`?

Reply:
Well in this case timeago only renders text so it would be more efficient to use the already installed moment. You could make a util out of it (the moment duration code is a bit long). Whatever you choose.

Reply:
I'll try to implement own component, and if it will become too massive - I'll leave `react-timeago`, otherwise I'll uninstall it.

Reply:
@ranbena getredash/redash#3385